tests/pep492/test_async_await.py

```diff
@@ -53,6 +54,30 @@
     assert cursor.closed
 
 
+@asyncio.coroutine
+async def test_pool_context_manager_timeout(pg_params, loop):
```

Comment:
@thehesiod I reworked test a bit, please review

Reply:
hmm, wonder how it worked locally, can't believe I checked it in with the debug code still in there, thanks!  looks good!

Reply:
Thank! Wanted double check this code.